UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Webjob.UnitTests/Services/FulfilmentDataServiceTest.cs

```diff
@@ -70,7 +70,11 @@ public void Setup()
                 Info = "INFO",
                 ProductFileName = "TEST.TXT",
                 CatalogFileName = "CATALOG.031",
-                CommentVersion = "VERSION=1.0"
+                CommentVersion = "VERSION=1.0",
+                ContentInfo =  "DVD INFO",
+                Content = "Catalogue",
+                CatalogueType = "ADC",
+                AdcFolderPath = "ADC"
```

Comment:
Can we have single key like "Adc": "Adc" and use it as required as same value in both Key ?

Reply:
Resolved